doc/getting-started/getting-started.rst

```diff
@@ -96,15 +97,22 @@ To set up the ACRN build environment on the development computer:
    <https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview>`__ to
    install a new OS on the development computer.
 
-#. Update Ubuntu with any outstanding patches, and install the necessary ACRN
-   build tools and dependencies:
+#. Update Ubuntu with any outstanding patches:
 
```

Comment:
Recommend formatting this step as substeps and write what's happening, rather than saying "followed by"